enableLogging macro not fully specifying namespace

When using the `enableLogging` macro outside of the `framework` namespace, one can see

```
/Users/einarelen/ldmx/ldmx-sw/Framework/include/Framework/Logger.h:137:11: error: 'logging' does not name a type; did you mean 'long'?
  137 |   mutable logging::logger theLog_{logging::makeLogger(name)};
      |           ^~~~~~~
/Users/einarelen/ldmx/ldmx-sw/Framework/include/Framework/Logger.h:137:11: note: in definition of macro 'enableLogging'
  137 |   mutable logging::logger theLog_{logging::makeLogger(name)};
      |           ^~~~~~~
```

This can be resolved simply by updating the macro to use a fully-specified namespace.

--- a/include/Framework/Logger.h
+++ b/include/Framework/Logger.h
@@ -134,7 +134,7 @@ void close();
  * in any class functions.
  */
 #define enableLogging(name) \
-  mutable logging::logger theLog_{logging::makeLogger(name)};
+  mutable ::framework::logging::logger theLog_{::framework::logging::makeLogger(name)};
 
 /**
  * @macro ldmx_log
@@ -142,6 +142,6 @@ void close();
  * Assumes to have access to a variable named theLog_ of type logger.
  * Input logging level (without namespace or enum).
  */
-#define ldmx_log(lvl) BOOST_LOG_SEV(theLog_, framework::logging::level::lvl)
+#define ldmx_log(lvl) BOOST_LOG_SEV(theLog_, ::framework::logging::level::lvl)
 
 #endif  // FRAMEWORK_LOGGER_H

